client-extensions/shared-projects/fds-cell-renderer/.gitignore

```diff
@@ -0,0 +1,5 @@
+/.liferay.json
```

Comment:
we may want to remove liferay.json, dist and build in the future if we can get rid of them bc we move everything to the yaml file.